### ISSUE TITLE
[Task] Add UI test for React view to Severless Artillery actions for `/users/{userid}/tasks/{taskid}` endpoint

### ISSUE THREAD

**Issue by nataliagourova** [North-Seattle-College/ad440-winter2021-thursday-repo#197]
**Is your feature request related to a problem? Please describe.**
We need to add UI testing framework for relevant React views.
slsart/ui/users/{userid}/tasks/{taskid}
Create a completely new action to do a Serverless Artillery test. 
Note: right now, the actions trigger in parallel, and we need to find way to trigger them in sequence, this will be handled in another issue

**Describe the solution you'd like**
A test that gives a baseline for the overhead on the endpoint's UI

**Describe alternatives you've considered**
Creating four tests as one action, for all four endpoints. It was decided against it to limit the test load on the machine, and distribute the tasks between the test members.

DATE | ACTIVITY | TIME
-- | -- | --
02/25 | Research on headless chrome and automation | 1 hour
03/25 | Puppeteer research | 1 hour
3/03 |GitHub Actions documentation review | 2 hours
3/03| Working GitHub Actions code files review |1 hour
3/03 | Write Code and debug | 3 hours
3/04 | AZ Blob connection string issues troubleshooting | 2 hours
  | Total | 10 hours

**Comment by nataliagourova** [comment]
Class notes: 
1. Create a separate GitHub Action for the UI test only. Name it "**UI Test /users/1/tasks/1**". 
Action file name should be **ui-tests-users.yml**
2. In the action load the UI in the testing framework https://nscstrdevusw2thucommon.z5.web.core.windows.net/users/1/tasks/1
3. Take a screenshot
4. Save it to azure Storage container https://nscstrdevusw2thucommon.blob.core.windows.net/screenshots 

*The UI actions should be triggered last (after the other actions).
/users
/users/{user_id}
/users/{user_id}/tasks
/users/{user_id}/tasks/{task_id}

**Comment by nataliagourova** [comment]
the Action is fired off on pr pushes successfully, failing at the AZ Blob Connection Point. Tested Azure Blob upload code on local machine with az cli successfully.
